tools/simpleui/main.go

```diff
@@ -68,7 +68,7 @@ type ui struct {
 }
 
 func main() {
-	conn, err := grpc.Dial("localhost:8080", grpc.WithInsecure(), grpc.WithBlock())
+	conn, err := grpc.Dial("localhost:50051", grpc.WithInsecure(), grpc.WithBlock())
```

Comment:
Why change the port number?

Reply:
I think this was changed in a different pr. So will update this. It happened due to rebasing.